tarodb/sqlc/queries/assets.sql

```diff
@@ -237,6 +237,20 @@ JOIN key_group_info_view
         sqlc.narg('key_group_filter') IS NULL)
 GROUP BY key_group_info_view.tweaked_group_key;
 
+-- name: FetchAssetsWithGroup :many
```

Comment:
The idea is that we don't need all the other information returned along side `QueryAssets` so we'd use this instead?

Reply:
Yes, the idea was that the RPC only needs to show the most human-readable info on each asset in a group, so we can drop the other fields.

At a minimum I think the query backing the RPC doesn't need any filters.